hardware_interface/include/hardware_interface/loaned_command_interface.hpp

```diff
@@ -81,12 +81,12 @@ class LoanedCommandInterface
     }
   }
 
-  const std::string get_name() const { return command_interface_.get_name(); }
+  const std::string & get_name() const { return command_interface_.get_name(); }
 
   const std::string & get_interface_name() const { return command_interface_.get_interface_name(); }
 
   [[deprecated(
-    "Replaced by get_name method, which is semantically more correct")]] const std::string
+    "Replaced by get_name method, which is semantically more correct")]] const std::string &
```

Comment:
does it make sense to change deprecated methods?

Reply:
You're absolutely right.
There's no need to make breaking changes for the deprecated methods, so I'll add a commit to revert the changes.